src/cart/crt_iv.c

```diff
@@ -2512,8 +2512,13 @@ handle_ivupdate_response(const struct crt_cb_info *cb_info)
 	/* For bi-directional updates, transfer data back to child */
 	if (iv_info->uci_sync_type.ivs_flags & CRT_IV_SYNC_BIDIRECTIONAL) {
 		transfer_back_to_child(&input->ivu_key, iv_info, true,
+<<<<<<< HEAD
+					cb_info->cci_rc);
+		goto exit;
+=======
```

Comment:
(style)  spaces required around that '==' (ctx:ExO)
(style)  spaces required around that '==' (ctx:OxO)
(style)  spaces required around that '==' (ctx:OxO)
(style)  spaces required around that '=' (ctx:OxE)